lib/core/src/lib/form/components/widgets/amount/amount.widget.spec.ts

```diff
@@ -203,7 +201,7 @@ describe('AmountWidgetComponent - rendering', () => {
         expect(await field.getPrefixText()).toBe('$');
 
         const widgetLabel = fixture.nativeElement.querySelector('label.adf-label');
-        expect(widgetLabel.textContent).toBe('Test Amount*');
+        expect(widgetLabel.textContent.trim()).toBe('Test Amount*');
```

Comment:
Formatting file shouldn't require any changes in unit tests

Reply:
So we should keep bad code formatting to keep unit tests as they are? Actually that is common thing in tests. I think this is good solution, because that spaces are not visible in DOM. Other solution are: breaking back code formatting or adding spaces to testing value - in my opinion both are worse solutions.